WebDriverAgentTests/IntegrationTests/XCUIApplicationHelperTests.m

```diff
@@ -40,7 +40,7 @@ - (void)testQueringSpringboard
 
 - (void)testApplicationTree
 {
-  XCTAssertNotNil(self.testedApplication.fb_tree);
+  XCTAssertNotNil([self.testedApplication fb_tree:[NSSet setWithArray:@[@"visible"]]]);
```

Comment:
unfortunately this test does not fully verify the new functionality. There must be an assertion that the attribute is not present in the resulting tree